src/manifest.safari.json

```
@@ -1,12 +1,12 @@
 {
   "manifest_version": 2,
   "author": "Ghostery",
-  "name": "__MSG_name__",
+  "name": "Ghostery - Ad Blocker",
```

Lets keep it simple for now
```suggestion
  "name": "Ghostery",
```